webapp/channels/src/components/password_reset_send_link/__snapshots__/password_reset_send_link.test.tsx.snap

```diff
@@ -1,60 +1,7 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`components/PasswordResetSendLink should match snapshot 1`] = `
-<div>
-  <BackButton
-    url="/"
-  />
-  <div
-    className="col-sm-12"
-  >
-    <div
-      className="signup-team__container"
-    >
-      <MemoizedFormattedMessage
-        defaultMessage="Password Reset"
-        id="password_send.title"
-        tagName="h1"
-      />
-      <form
-        onSubmit={[Function]}
-      >
-        <p>
-          <MemoizedFormattedMessage
-            defaultMessage="To reset your password, enter the email address you used to sign up"
-            id="password_send.description"
-          />
-        </p>
-        <div
-          className="form-group"
-        >
-          <LocalizedPlaceholderInput
-            autoFocus={true}
-            className="form-control"
-            id="passwordResetEmailInput"
-            name="email"
-            placeholder={
-              Object {
-                "defaultMessage": "Email",
-                "id": "password_send.email",
-              }
-            }
-            spellCheck="false"
-            type="email"
-          />
-        </div>
-        <button
-          className="btn btn-primary"
-          id="passwordResetButton"
-          type="submit"
-        >
-          <MemoizedFormattedMessage
-            defaultMessage="Reset my password"
-            id="password_send.reset"
-          />
-        </button>
-      </form>
-    </div>
-  </div>
-</div>
+<ContextConsumer>
+  <Component />
+</ContextConsumer>
```

Comment:
Could we make this snapshot match what it was before by going one level deeper with it? I think we might be able to do that with `shallow(...).dive()`